### PR TITLE
gnome-weather: update to 3.32.0.

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by Cogitri** [comment]
SIGSEVs for me as soon as I try using a custom location, will message upstream.

**Comment by Cogitri** [comment]
https://gitlab.gnome.org/GNOME/gnome-weather/issues/26

File: srcpkgs/gnome-weather/template [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-lib32disabled=yes
-build_style=gnu-configure
-configure_args="--disable-schemas-compile"
-hostmakedepends="pkg-config intltool itstool gnome-doc-utils glib-devel
- $(vopt_if gir gobject-introspection)"
+build_helper="gir"
+build_style=meson
+hostmakedepends="pkg-config itstool glib-devel gjs"
 makedepends="gtk+3-devel gjs-devel libgweather-devel geoclue2-devel"
 depends="desktop-file-utils"
```
Comment (maxice8) [inline srcpkgs/gnome-weather/template]: missing gjs, geoclue2, libgweather, gnome-desktop on the dependencies.

There are no shlibs that can be used to find dependencies properly. -1 for gjs

**Comment by Cogitri** [comment]
Alright, the crashes fixed themselves, so seems to good merge now.

File: srcpkgs/gnome-weather/template (hunk 1 above)
Comment (maxice8) [reply srcpkgs/gnome-weather/template]: Merged manually with these added dependencies or the package doesn't start when installed without having the full GNOME stack installed.

**Pull request closed without merging**